### ISSUE TITLE
about capability_type

### ISSUE THREAD

**Issue by simongcc** [emfluenceindia/gutenberg-custom-field-example#2]
Hello! @emfluenceindia  Thanks for your sharing!

I saw that your post type registration is using an argument "compatibility_type".
Did you mean "capability_type"?

Although currently, the post type is still showing up. But after a new book is created, the page cannot be found if visitng the link.

I tried to confirm with the [source code](https://core.trac.wordpress.org/browser/tags/5.0.2/src/wp-includes/post.php#L0) and can only find the "capability_type";

**Comment by emfluenceindia** [comment]
Thank you for taking time. I will look into it asap. But why you are not being able to see the post is hard to understand because at my end it is absolutely fine. I will get back to you soon :)

**Comment by emfluenceindia** [comment]
You are right! It was a mistake in my part. It has to be `capability_type`. I would be thankful if you please fix it and create a PR. I won;t have access to my computer for next 10 days.

However, loading book post page is absolutely fine at my end.

Thank you for your help!
